### PR TITLE
Update description

### PR DESCRIPTION
**Pull request by rviscomi**
Fixes https://github.com/HTTPArchive/httparchive.org/issues/629

### CODE REVIEW

**Comment by tunetheweb** [comment]
Why dropping of "The"? It's what's on our [home page](https://httparchive.org/):

image

Reads weird to me without it.

**Comment by rviscomi** [comment]
Reverted "the"

File: templates/about.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {% block title %}About HTTP Archive{% endblock %}
```
Comment (tunetheweb) [inline templates/about.html:4]:
```suggestion
{% block title %}About the HTTP Archive{% endblock %}
```

Should we review all the titles?

File: templates/index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 {% block title %}HTTP Archive{% endblock %}
```
Comment (tunetheweb) [inline templates/index.html:4]:
```suggestion
{% block title %}The HTTP Archive{% endblock %}
```

**Review by tunetheweb — CHANGES_REQUESTED** [review]

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**